Servers/UI/OJS.Servers.Ui/ClientApp/src/components/guidelines/buttons/Button.tsx

```diff
@@ -64,6 +65,7 @@ const classNameToButonType = {
     [ButtonType.image]: styles.image,
     [ButtonType.toggled]: styles.toggled,
     [ButtonType.untoggled]: styles.untoggled,
+    [ButtonType.neutral]: styles.neutral,
```

Comment:
What is the difference between button type plain and neutral?

Reply:
they are similar, but plain one does not look like the one in the design, so I added a neutral one, didnt want to. change the previous one since it is used in a few places

Reply:
If the difference is minor, we should use what we already have, not add another type. Might be unintentional difference introduced from the designer.

Reply:
plain is just the text, without border, it doesn't really look like a button, to me it makes sense to use plain in another kind of situations, while neutral (with the design required for this button) itself is present in the contest submission page as well